vendor/github.com/jstemmer/go-junit-report/formatter/BUILD

```diff
@@ -2,15 +2,11 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 
 go_library(
     name = "go_default_library",
-    srcs = [
-        "doc.go",
-        "unstructured.go",
-        "value.go",
-    ],
-    importmap = "k8s.io/kubernetes/vendor/sigs.k8s.io/structured-merge-diff/value",
-    importpath = "sigs.k8s.io/structured-merge-diff/value",
+    srcs = ["formatter.go"],
```

Comment:
This does not look like a right file move

Reply:
This is auto generated by hack/update-vendor.sh. the correct process is remove "vendor/sigs.k8s.io/structured-merge-diff/value/BUILD". and new added file "vendor/github.com/jstemmer/go-junit-report/formatter/BUILD". not sure how git display this as file moving.